devicon.json

```
@@ -1,4 +1,21 @@
 [
+    {
+      "name": "adonisjs",
+      "tags": [
+          "nodejs",
+          "framework"
+      ],
+      "versions": {
+          "svg": [
+              "original"
+          ],
+          "font": [
+              "plain"
```

There's no plain version, so we should use the `original` version instead and add it as an alias.

```suggestion
              "original"
```

---

Sorry, overlooked it :)